dinghy-lib/src/ssh/device.rs

```diff
@@ -60,7 +61,8 @@ impl SshDevice {
     }
 
     fn sync<FP: AsRef<Path>, TP: AsRef<Path>>(&self, from_path: FP, to_path: TP) -> Result<()> {
-        let mut command = Command::new("/usr/bin/rsync");
+        let mut command = Command::new(env::var("DINGHY_LOCAL_RSYNC_PATH").unwrap_or("/usr/bin/rsync".to_string()));
```

Comment:
extract `env::var("DINGHY_LOCAL_RSYNC_PATH").unwrap_or("/usr/bin/rsync".to_string())` to a var ?

Reply:
nvm, those are two different vars